lxd/storage/drivers/driver_lvm.go

```diff
@@ -501,8 +501,6 @@ func (d *lvm) Mount() (bool, error) {
 			return false, err
 		}
 		defer loopFile.Close()
-
-		return true, nil
```

Comment:
@stgraber I noticed that this will change the return behaviour of calling `Mount()` on a loopback storage pool because it will now never return `true` for "our mount" indicator. This value is used in a few places, such as the storage patch to rename custom volumes and when creating a storage pool, it is unmounted after create as per the specification.

That being said, it also never returned true for non-loopback pools when activating the volumes in the volume group, as the concept was more tricky to bundle into a true/false return value (this is because a volume group can be "partially" activated by having some of its volumes activated and others not.

However this change does mean that the loop files will be left open if Mount has opened them, because our mount is returned as false always now. I don't think this will cause a problem now, but may cause unexpected issues in the future.

This is related to our discussion in https://github.com/lxc/lxd/pull/7037#issuecomment-600611390 where we talked about activating and deactivating the volume groups themselves, rather than the volume group (which just activates all volumes in the group anyway). This would allow us to make the pool `Mount()` return true only if the loopback file was activated.